bitrise.yml

```diff
@@ -1250,8 +1261,9 @@ workflows:
             $BITRISE_SOURCE_DIR/gradlew ":trace-sdk:publishToMavenLocal"
             $BITRISE_SOURCE_DIR/gradlew -p "./trace-gradle-plugin/" publishToMavenLocal --stacktrace
             $BITRISE_SOURCE_DIR/gradlew -p "./trace-gradle-plugin/" build --stacktrace
+            $BITRISE_SOURCE_DIR/gradlew "tagModule -PmoduleToTag=trace-gradle-plugin"
             $BITRISE_SOURCE_DIR/gradlew -p "./trace-gradle-plugin/" publish --stacktrace
-        title: build_trace-gradle-plugin
+        title: build_&_publish_trace-gradle-plugin
```

Comment:
I am not sure, what is the case when it is easier to search and find?

I choose this because it is more compact and visible.

Reply:
hmm. so our yml file is very large, i usually search by title to find the code block i want to update - i don't think i'd search with special characters e.g. &, and i'd type the word and this would mean it wouldn't find the section.

I think it's also setting a precedent, and i don't think we shouldn't allow special characters in the title (aside from _ and -). I think it'd be fine if it were the description.

Reply:
I feel this reason depends on habits of usage As this started a nit I would leave it now as is, unless if there is any other strong reason to not leave it? Luckily it has quite low impact, and can be addressed anytime if we make up our minds.

Reply:
yeah, I understand, it was always a nit :)